frog/imports/ui/GraphEditor/index.js

```diff
@@ -28,28 +28,35 @@ export default class AppClass extends Component {
 }
 
 const keyDown = e => {
+  if (store.mode === 'rename') {
+    return;
+  }
   if (
     !store.ui.overGraph ||
     store.state.mode === 'rename' ||
     e.ctrlKey ||
     e.altKey ||
-    e.metaKey ||
-    e.shiftKey
+    e.metaKey
   ) {
     return;
   }
 
   if (e.keyCode === 27) {
-    // esc
+    // esc - cancel selection
     e.preventDefault();
     store.ui.cancelAll();
     store.ui.unselect();
   }
   if (e.keyCode === 8) {
-    // backspace
+    // backspace - delete
     e.preventDefault();
     store.deleteSelected();
   }
+  if (e.keyCode === 191 && e.shiftKey) {
```

Comment:
Is there an other way than using `keyCode` ? On my swiss keyboard, the help opens if I click on the key showing `§` and `°` and not `?`. Works with the expected key if I switch to English keyboard. 

Suggested fixes:
- Open the help with a button instead of a keyboard shortcut
- use the letter `H`
- see what we can do with `charCode` instead of `keyCode`